apps/meteor/ee/server/api/sessions.ts

```diff
@@ -0,0 +1,181 @@
+import { check, Match } from 'meteor/check';
+import { IUser } from '@rocket.chat/core-typings';
+
+import { isSessionsProps } from '../../definition/rest/v1/sessions/SessionsProps';
+import { Users, Sessions } from '../../../app/models/server/raw/index';
+import { API } from '../../../app/api/server/api';
+import { hasLicense } from '../../app/license/server/license';
+
+API.v1.addRoute(
+	'sessions/list',
+	{ authRequired: true },
```

Comment:
This is the only handler here using Check instead of AJV 🙈

Reply:
No issues